setuptools/command/install_scripts.py

```diff
@@ -37,6 +38,10 @@ def run(self):
         if is_wininst:
             exec_param = "python.exe"
             writer = ei.WindowsScriptWriter
+        if exec_param == sys.executable:
```

Comment:
The whole point of creating the CommandSpec class was to try to encapsulate as much of these decision points as possible, so it's sad to see this one re-introduced. What's worse is that it seems to be special-casing a special case, inferring that if the string is `sys.executable`, then it was meant to be escaped, but in any other case it's not. This expectation would do the wrong thing if someone passed an executable with spaces that wasn't sys.executable (e.g. "C:\My Python Env\python.exe" where sys.executable is "C:\Python35\python.exe"). I'm not sure if there's something better do be done other than grin and bear it.